randovania/games/super_metroid/pickup_database/pickup-database.json

```diff
@@ -269,7 +330,9 @@
             "pickup_category": "morph_ball",
             "broad_category": "morph_ball_related",
             "model_name": "Spring Ball",
-            "offworld_models": {},
+            "offworld_models": {
+                "am2r": "sItemJumpBall",
```

Comment:
I might suggest using the morph model for prime and dread

Reply:
This was copied from am2r's implementation, should I go add morph model there too then?